scripts/setup.py

```diff
@@ -13,6 +13,7 @@
 import configuration
 import constants
 import mykb
+import cups
```

Comment:
Add a try/except/pass block to avoid breaking usage for people that don't need printing feature and haven't installed pycups